core/src/View/MDSpan/Kokkos_MDSpan_Accessor.hpp

```diff
@@ -34,35 +34,30 @@ namespace Impl {
 
 template <class MemorySpace, class NestedAccessor>
 struct SpaceAwareAccessor {
+  // Part of Accessor Requirements
   using element_type     = typename NestedAccessor::element_type;
   using reference        = typename NestedAccessor::reference;
   using data_handle_type = typename NestedAccessor::data_handle_type;
-
   using offset_policy =
       SpaceAwareAccessor<MemorySpace, typename NestedAccessor::offset_policy>;
 
-  using memory_space = MemorySpace;
+  // Specific to SpaceAwareAccessor
+  using memory_space         = MemorySpace;
+  using nested_accessor_type = NestedAccessor;
```

Comment:
I don't necessary have an issue with adding that public member type but you don't say we might want to have it and it is not used.

Reply:
I still think we should have it, and I did add the nested_accessor() function as in https://eel.is/c++draft/linalg.scaled

Reply:
`std::linalg::scaled_accessor` does not define a member type for it.  Anyway, I don't feel too strongly about it.

Reply:
Yeah I know, the comma in my reply was basically saying "I do think we should have the typedef" and orthogonal to that "I also added the nested_accessor() function as in scaled_accessor"